tests/infra_test.go

```diff
@@ -725,42 +729,21 @@ var _ = Describe("[Serial]Infrastructure", func() {
 			}
 		})
 
-		It("[test_id:4143]should include the network metrics for a running VM", func() {
-			metrics := collectMetrics("kubevirt_vmi_network_")
+		table.DescribeTable("should include metrics for a running VM", func(metricSubstring, operator string) {
+			metrics := collectMetrics(metricSubstring)
 			By("Checking the collected metrics")
 			keys := getKeysFromMetrics(metrics)
 			for _, key := range keys {
 				value := metrics[key]
-				Expect(value).To(BeNumerically(">=", float64(0.0)))
+				fmt.Fprintf(GinkgoWriter, "metric value was %f\n", value)
+				Expect(value).To(BeNumerically(operator, float64(0.0)))
```

Comment:
ditto

Reply:
didnt touch this line, just indented,
also in case of error the string would be redundant because the error will say it already

Reply:
the failed test would say "expected 2.0 to be numerically 0.0", I want it to say something more meaningful, if possible.

Reply:
its not part of this PR scope, we can do it if needed on other PR please.
also since the test itself say what is the metric that is checked, this will be redundant, all the info is there, just few lines above in the log block.

Reply:
fair enough